lib/fastlane/plugin/maestro_orchestration/actions/maestro_orchestration_android_action.rb

```diff
@@ -83,11 +95,10 @@ def self.demo_mode(params)
         sh("#{params[:sdk_dir]}/platform-tools/adb shell am broadcast -a com.android.systemui.demo -e command network -e mobile show -e datatype none -e level 4")
       end
 
-      def self.build_and_install_android_app(params)
-        UI.message("Building Android app...")
-        other_action.gradle(task: "assembleDebug")
+      def self.install_android_app(params)
+        UI.message("Installing Android app...")
 
-        apk_path = Dir["app/build/outputs/apk/debug/app-debug.apk"].first
+        apk_path = Dir["app/build/outputs/apk/release/app-release.apk"].first
```

Comment:
I missed this first time, but one of our apps doesn't output with name `app-release`, but `projectname` and some of our apps build aabs (not apks).